container-runtime/gvisor-kvm/gvisor-kvm.part

```diff
@@ -0,0 +1,6 @@
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runsc-kvm]
```

Comment:
if the extension just adds this file, can we simply merge it back with `gvisor` and let users choose via the runtime class?

Reply:
Mhm, that's a good point, that makes it even simpler!